src/MembershipManager.php

```diff
@@ -189,6 +189,9 @@ public function getGroupIds(EntityInterface $entity, $group_type_id = NULL, $gro
       $query = $this->entityTypeManager
         ->getStorage($target_type)
         ->getQuery()
+        // When rebuilding node_access grants then checking node_access is
```

Comment:
one more thought - I think we can change the function signature to `public function getGroupIds(EntityInterface $entity, $group_type_id = NULL, $group_bundle = NULL, $check_access = FALSE) {`